apps/skateboard/web/elm/Letter.elm

```diff
@@ -0,0 +1,116 @@
+module Letter (..) where
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import List exposing (..)
+import Array exposing (..)
```

Comment:
Do you think it's worth making this explicit like GameBoard.elm? (and all maybe except the html and html.attributes)